client/app/components/calendar/event/EventWrapper.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint react/prefer-stateless-function: 0 */
+
+import React from 'react';
+import dimension from 'react-dimensions';
+import { If, Then } from 'react-if';
+
+import EventGroup from './EventGroup';
+
+class EventWrapper extends React.Component {
+  static propTypes = {
+    events: React.PropTypes.array
+  };
+
+  render() {
+    return (
+      <If condition={this.props.events != null}>
+        <Then>{() =>
```

Comment:
Yes because a child of `If` must be a `ReactElement` and it has to be a function because it cannot get evaluated when the condition is false because it will throw an error 'cuz `this.props.events` is undefined